docs/networking/clusternetwork.md

```diff
@@ -63,15 +63,15 @@ You are allowed to add the custom cluster network, which will not be available u
 
    ![](/img/v1.1/networking/create-clusternetwork.png)
 
-2. Click the **Create Network Config** button on the left of the cluster network to create a new network configuration.
+2. Click the **Create Network Config** button on the right of the cluster network to create a new network configuration.
 
    ![](/img/v1.1/networking/create-network-config-button.png)
 
 3. In the **Node Selector** tab, specify the name and choose one of the three methods to select nodes where the network configuration will apply. If you want to cover the unselected nodes, you can create another network configuration.
 
    ![](/img/v1.1/networking/select-nodes.png)
 
-4. Click the **Uplink** tab to add the NICs, and configure the bond options and link attributes.
+4. Click the **Uplink** tab to add the NICs, and configure the bond options and link attributes. The bond mode defaults to `active-backup`.
```

Comment:
L#80:
`one of the there selected nodes`  or `one of the trhee selected nodes`
`there` or `three` ?